apps/src/code-studio/components/progress/course_progress.jsx

```diff
@@ -22,14 +24,25 @@ const styles = {
  */
 const CourseProgress = React.createClass({
   propTypes: {
+    onOverviewPage: React.PropTypes.bool.isRequired,
```

Comment:
My personal opinion is that `onOverviewPage` is clearer (in part because I don't really think of these as overviewControls). It also has the benefit of being the same name used in progress.js